x/staking/keeper/keeper.go

```diff
@@ -442,7 +442,8 @@ func (k *Keeper) IterateStakingSequencesAndApplyFn(ctx sdk.Context, f func(seque
 // Slashing api's
 // AddValidatorSigningInfo creates a signing info for validator
 func (k *Keeper) AddValidatorSigningInfo(ctx sdk.Context, valID hmTypes.ValidatorID, valSigningInfo hmTypes.ValidatorSigningInfo) error {
-	k.ModuleCommunicator.CreateValiatorSigningInfo(ctx, valID, valSigningInfo)
+	// todo: enable after slashing module added
```

Comment:
what needs to be done here?

Reply:
https://github.com/maticnetwork/heimdall/blob/e58da51d56857bd3d2f61c8e7b22f4ff024bac75/app/app.go#L182-L186